From 0b2d302485c4b16435fcbcc2a97a0a5aefc59614 Mon Sep 17 00:00:00 2001
From: Gatlin Cheyenne Johnson <gatlin@niltag.net>
Date: Sat, 19 Nov 2022 09:55:19 -0600
Subject: [PATCH] w3m image support

---
 x.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/x.c b/x.c
index 8a16faa..768fe93 100644
--- a/x.c
+++ b/x.c
@@ -1626,6 +1626,9 @@ xsettitle(char *p)
 int
 xstartdraw(void)
 {
+  if (IS_SET(MODE_VISIBLE)) {
+    XCopyArea(xw.dpy, xw.win, xw.buf, dc.gc, 0, 0, win.w, win.h, 0, 0);
+  }
 	return IS_SET(MODE_VISIBLE);
 }
 
-- 
2.35.1

